gateway/ln-gateway/src/lightning/mod.rs

```diff
@@ -222,7 +222,7 @@ impl dyn ILnRpcClient {
 
 #[derive(Serialize, Deserialize, Debug)]
 pub struct ChannelInfo {
-    pub remote_pubkey: String,
+    pub remote_pubkey: Option<secp256k1::PublicKey>,
```

Comment:
Why is it an option? I see that parsing might fail, but that could probably be handled with an `expect` since it means that the LN service is returning invalid data.

Reply:
@tvolk131

Reply:
I'm fine removing the `Option`! Done.